docs/src/components/ComponentDoc/ComponentDocTag.tsx

```diff
@@ -1,27 +1,82 @@
 import * as React from 'react'
 import * as _ from 'lodash'
 import { Header, Message } from 'semantic-ui-react'
+const behaviorMenuItems = require('docs/src/behaviorMenu')
 
 const headerStyle = {
   whiteSpace: 'pre-line',
 }
 
+interface DefaultBehaviorInfo {
+  name: string
+  url: string
+}
+
 class ComponentDocTag extends React.Component<any, any> {
   getTagDescription = (forTag, fromInfo) => {
     const tags = (fromInfo.docblock && fromInfo.docblock.tags) || []
     return _.result(_.find(tags, 'title', forTag), 'description')
   }
 
+  findDefaultBehaviorInfo = (fetchDefaultBehaviorInfo): DefaultBehaviorInfo => {
+    const accessibilityProperty = fetchDefaultBehaviorInfo.props.find(
+      property => property.name === 'accessibility',
+    )
+    if (!accessibilityProperty) {
+      return undefined
+    }
+
+    const accPropertyFormatted = accessibilityProperty.defaultValue.split('.').pop() + '.ts'
+
+    for (const behavior of behaviorMenuItems) {
+      const variation = behavior.variations.find(v => v.name === accPropertyFormatted)
+      if (variation) {
+        return this.createDefaultBehaviorInfo(behavior, variation)
+      }
+    }
+
+    return undefined
+  }
+
+  createDefaultBehaviorInfo(behavior: any, variation: any): DefaultBehaviorInfo {
+    const behaviorsFolder = behavior.type + 's'
+    const defaultBehaviorFile = variation.name.replace('.ts', '')
+    return {
+      name: `${defaultBehaviorFile}`,
+      url: `${behaviorsFolder}/${behavior.displayName}#${_.kebabCase(defaultBehaviorFile)}`,
+    }
+  }
+
+  renderDefaultBehavior(info) {
+    const defaultBehaviorInfo: DefaultBehaviorInfo = this.findDefaultBehaviorInfo(info)
+    if (defaultBehaviorInfo) {
+      return (
+        <Header.Subheader>
+          <span>
+            Default behavior: <a href={defaultBehaviorInfo.url}> {defaultBehaviorInfo.name} </a>
+          </span>
+        </Header.Subheader>
+      )
+    }
+
+    return undefined
+  }
+
   render() {
     const { info, tag, title, errorMessage } = this.props
-    const description = this.getTagDescription(tag, info) || (
-      <Message error content={errorMessage} compact={true} />
-    )
+    const accDescription = this.getTagDescription(tag, info)
+    const description =
+      accDescription || this.findDefaultBehaviorInfo(info) ? (
```

Comment:
so, we are not using the result of `findDefaultBehaviorInfo `? seems that we should call `renderDefaultBehavior()` here directly - otherwise we are calling the same method twice with no good reason for that

Reply:
in this part of the code we just checking if there is any description or there should be error message. This part of structure I took from origin code. 

We are using the result in this condition, because if there is no default behavior, then the method findDefaultBehaviorInfo() returns undefined...

About calling method twice. It is correct. Maybe we could save result of the method in some global variable and then when we will call it second time and global variable will exists, then we do not need to call it again. What do you think about?